media-gfx/pngnq/files/pngnq-1.1-gcc14-build-fix.patch

```diff
@@ -0,0 +1,33 @@
+Bug: https://bugs.gentoo.org/919210
```

Comment:
Have you tried to upstream this patch?

Reply:
Yep, I've also updated the patch with the link.

https://sourceforge.net/p/pngnq/patches/7/